lib/context2d.js

```diff
@@ -16,10 +16,6 @@ var canvas = require('./bindings')
   , CanvasPattern = canvas.CanvasPattern
   , ImageData = canvas.ImageData;
 
-var parseCssFont = require('parse-css-font');
```

Comment:
I didn't realise that it would be part of the pull request.. that was not intentional.